app/controllers/slot_ranges_controller.rb

```diff
@@ -18,13 +18,19 @@ def update
       @slot_range.slots.destroy_all
       @slot_range.update(slot_range_parameters)
     end
-    redirect_to edit_user_path(@guider), flash: { success: "Guider \"#{@guider.name}\" has been updated" }
+    redirect_to(
+      edit_user_path(@guider),
+      success: "Slot Range \"#{SlotRangePresenter.new(@slot_range)}\" has been updated"
```

Comment:
Can we just use 'The slot range has been updated' then or similar. i.e. don't refer to any of the actual slot range particulars?